src/api-service/__app__/onefuzzlib/workers/nodes.py

```diff
@@ -35,7 +35,7 @@
 from .shrink_queue import ShrinkQueue
 
 NODE_EXPIRATION_TIME: datetime.timedelta = datetime.timedelta(hours=1)
-NODE_REIMAGE_TIME: datetime.timedelta = datetime.timedelta(days=7)
+NODE_REIMAGE_TIME: datetime.timedelta = datetime.timedelta(days=6)
```

Comment:
The queries use this value to define stale nodes. 
![image](https://user-images.githubusercontent.com/74685766/151222985-bac6d23b-548a-41f6-ab8d-a0d5459562c5.png)

Reply:
Unless you just mean changing from 7 to 6. In that case, I'd say no. I'm changing the value to 6 because some of our s360 issues are based off 7 day timers. For example, antimalware asks us to update our signatures every 7 days - which we do via reimage. 
Since the s360 timer is on 7 days, it makes sense for us to have our timer be one day less to avoid weird overlap.